reference/halyard/custom.md

```diff
@@ -42,6 +42,18 @@ information.
 You can validate that your Custom Profiles have been picked up and applied to
 their services by checking `~/.hal/$DEPLOYMENT/history/service-profiles.yml`.
 
+### Custom Profile for Deck
+
+To supply custom settings or new values that override generated settings for the
+Deck service, you can place them in a file called `settings-local.js`. For
+example, to enable the `artifactsRewrite` feature flag, you can create the
```

Comment:
@fifthposition 
Is this flag going to live on, or is it just temporary?

Reply:
It's temporary (hopefully going away in 1.15, when hopefully these changes will be default).

Reply:
When that happens, will you or I remember to update this?

Reply:
If we document the `artifactsRewrite` changes as being behind `artifactsRewrite`, then I'll remember. :)